model/src/main/kotlin/OrtResult.kt

```diff
@@ -377,7 +377,7 @@ data class OrtResult(
     }
 
     /**
-     * True if the given [project] is excluded.
+     * True if the given [Project] is excluded.
```

Comment:
Shouldn't this somehow reference `[id]`?

Reply:
Right, I have added a commit to address that.